romana-install/group_vars/vagrant

```diff
@@ -1,7 +1,27 @@
 ram_size:
   controller:
     devstack: 4096
-    kubernetes: 1024
+    kubernetes: 2048
   compute:
     devstack: 1024
```

Comment:
We'd all need new laptops.

Controller needs 4GB to be functional. 4 + 2 = 6GB.
Running on a host with 8GB, there's not enough left for the host.